create_database.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+# module CreateDatabase
+module CreateDatabase
```

Comment:
why CreateDatabase? as I see, your module load, generate, delete data, not only create

Reply:
corrected. thanks